tools/harness-sniffer/OT_Sniffer.py

```diff
@@ -0,0 +1,135 @@
+'''OpenThread Sniffer API implementation'''
+
+import subprocess
+from GRLLibs.UtilityModules.ModuleHelper import ModuleHelper
+from ISniffer import ISniffer
+
+
+class OT_Sniffer(ISniffer):
+    common_speed = [460800, 115200, 9600]
+
+    def __init__(self, **kwargs):
+        try:
+            self.channel = kwargs.get('channel')
+            self.port = kwargs.get('addressofDevice')
+            self.baudrate = None
+            self.subprocess = None
+            self.is_active = False
+
+        except Exception, e:
+            ModuleHelper.WriteIntoDebugLogger('OT_Sniffer: [intialize] --> ' + str(e))
+
+    def discoverSniffer(self):
+        sniffers = []
+
+        p_discover = subprocess.Popen('extcap_ot.bat --extcap-interfaces', stdout=subprocess.PIPE, shell=True)
+        for line in p_discover.stdout.readlines():
```

Comment:
nit: `.readlines()` is not necessary.

Reply:
p.stdout return a file object, I think I should append '.readlines()'?

Reply:
you can try when you have time

Reply:
There will be problems after deleting the `.readlines()`